spec/spec_helper.rb

```diff
@@ -1,2 +1,6 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'hawk'
+require 'byebug'
```

Comment:
Sure, no problem. Actually I prefer pry, too, but had seen byebug in the other repos and thought it was more popular here